extensions/interactions/InteractiveMap/directives/interactive_map_interaction_directive.html

```diff
@@ -1,13 +1,5 @@
 <style>
-  /* Fixes for Google Maps zoom controls. */
-  .gm-style img {
-    max-width: none;
-  }
-  .gm-style label {
-    width: auto; display:inline;
-  }
-
-  #map_canvas {
+  #interactiveMap {
```

Comment:
I worry slightly about this. Should we use a CSS class for the style, and additionally generate a random ID so that each map instance is unique? I'm not sure.

Reply:
Added class. (Done!)